lib/Digitick/Sepa/DomBuilder/BaseDomBuilder.php

```diff
@@ -60,7 +60,6 @@ function __construct($painFormat)
         $this->root = $this->doc->createElement('Document');
         $this->root->setAttribute('xmlns', sprintf("urn:iso:std:iso:20022:tech:xsd:%s", $painFormat));
         $this->root->setAttribute('xmlns:xsi', "http://www.w3.org/2001/XMLSchema-instance");
-        $this->root->setAttribute('xsi:schemaLocation', "urn:iso:std:iso:20022:tech:xsd:$painFormat $painFormat.xsd");
```

Comment:
Ok, I'm not sure at all that is correct without the tag. https://wiki.xmldation.com/Support/Nordea/SchemaLocation

I have checked without the attribute in multiple banks and works fine.

Best thought, I think we should leave that attribute.